cdc/sink/ddlsink/mysql/async_ddl.go

```diff
@@ -15,25 +15,29 @@ package mysql
 
 import (
 	"context"
-	"database/sql"
 	"fmt"
 	"time"
 
 	"github.com/pingcap/log"
+	"github.com/pingcap/tidb/dumpling/export"
 	timodel "github.com/pingcap/tidb/pkg/parser/model"
 	"github.com/pingcap/tiflow/cdc/model"
-	"github.com/pingcap/tiflow/pkg/errors"
 	"go.uber.org/zap"
 )
 
+const timeout = 5 * time.Second
```

Comment:
It looks this part changes not relevant to the vector type. It's suggested revert this part, and create a new PR to do this.

Reply:
It is related to https://github.com/pingcap/tiflow/issues/11521, I will revert it before merging.